diana-api/src/main/java/org/apache/diana/api/reader/EnumReader.java

```diff
@@ -14,15 +14,15 @@
  *
  * @author Otávio Santana
  */
-public final class EnumReader implements ReaderField {
+public final class EnumReader implements ReaderField<Enum> {
```

Comment:
The Problem is: How can I now the enum type?
Because, I need the class information to read all enum attributes. 
Following this way, I need to create a ReaderField implementation to each enum, what is not good.